api/geth_backend.go

```diff
@@ -416,6 +444,30 @@ func (b *GethStatusBackend) OverwriteNodeConfigValues(conf *params.NodeConfig, n
 	return conf, nil
 }
 
+func (b *GethStatusBackend) updateAccountColorHashAndColorID(keyUID string, accountsDB *accounts.Database) (*multiaccounts.Account, error) {
```

Comment:
Might make sense to pass a multiAccount as a param instead of `keyUID`, as in 2 places this function is invoked, we have the multiAccount already, so this would save a DB call. Also, as the 2 invocations already include a conditional on `ColorHash` and `ColorID`, maybe we can also remove the superfluous conditional in the function body itself (line #452)